stdlib/3/faulthandler.pyi

```diff
@@ -15,7 +15,7 @@ else:
 def cancel_dump_traceback_later() -> None: ...
 def disable() -> None: ...
 def dump_traceback(file: _File = ..., all_threads: bool = ...) -> None: ...
-def dump_traceback_later(timeout: int, repeat: bool = ..., file: _File = ..., exit: bool = ...) -> None: ...
+def dump_traceback_later(timeout: Union[float, int], repeat: bool = ..., file: _File = ..., exit: bool = ...) -> None: ...
```

Comment:
```suggestion
def dump_traceback_later(timeout: float, repeat: bool = ..., file: _File = ..., exit: bool = ...) -> None: ...
```

Since `Union[int, float]` and `float` are identical for argument types.

Reply:
Thanks!